pkg/controller/kogitoapp/resource/deployment_config.go

```diff
@@ -197,7 +197,7 @@ func setReplicas(kogitoApp *v1alpha1.KogitoApp, dc *appsv1.DeploymentConfig) {
 // SetInfinispanEnvVars sets Infinispan variables to the given KogitoApp instance DeploymentConfig by reading information from the KogitoInfra
 func SetInfinispanEnvVars(cli *client.Client, kogitoInfra *v1alpha1.KogitoInfra, kogitoApp *v1alpha1.KogitoApp, dc *appsv1.DeploymentConfig) error {
 	if dc != nil && kogitoApp != nil &&
-		(kogitoInfra != nil && &kogitoInfra.Status != nil && &kogitoInfra.Status.Infinispan != nil) {
```

Comment:
can this removal cause any other kind of issue? a false positive or something?

Reply:
Not sure, but I think I can replace this with `kogitoInfra != nil && len(kogitoInfra.Status.Infispan.Condition) > 0`

Reply:
Checking other conditions as well, I think that should not give us any false positive

Reply:
Won't cause an issue removing those since this branch wasn't working any way. Interfaces can not be nil.